File: GerenciaMusic360.Entities/ComposerDetail.cs
using System;

namespace GerenciaMusic360.Entities
{
    public class ComposerDetail
    {
        public int Id { get; set; }
        public short AssociationId { get; set; }
        public short EditorId { get; set; }
        public int ComposerId { get; set; }
        public string IPI { get; set; }
        public DateTime? DateStart { get; set; }
        public DateTime? DateEnd { get; set; }

        public virtual Association Association { get; set; }
        public virtual Editor Editor { get; set; }

    }
}

File: GerenciaMusic360/ClientApp/app/shared/models/composer-detail.ts
import { IAssociation } from "./association";
import { IEditor } from "./editor";

export interface IComposerDetail {
    id: number;
    associationId: number;
    editorId: number;
    composerId: number;
    ipi: string;
    dateStart?: string;
    dateEnd?: string;
    association?: IAssociation;    
    editor?: IEditor;

}
